### PR TITLE
Update bicep from build templates

### PR DESCRIPTION
**Pull request by jongio**
Here are some optimizations and cleanup from project we created for build.

1. Sorted params added descriptions
2. Added support for ACA Addons with the ServiceType and ServiceBinds properties
3. Allow user to choose to not include dashboard in app insights
4. Allow user to create secrets in bulk

### CODE REVIEW

File: templates/common/infra/bicep/core/security/keyvault-secrets.bicep
Comment (wbreza) [inline templates/common/infra/bicep/core/security/keyvault-secrets.bicep:9]: Why are we setting batchsize to 1 here? Wouldn't we rather create them in parallel to speed up deployment?

File: templates/common/infra/bicep/core/host/container-app.bicep
Comment (wbreza) [inline templates/common/infra/bicep/core/host/container-app.bicep:43]: This is part of the ingress section and actually means whether the ingress is exposed externally or not.

File: templates/common/infra/bicep/core/host/container-app-upsert.bicep
Comment (wbreza) [inline templates/common/infra/bicep/core/host/container-app-upsert.bicep:61]: Same as other comment regarding whether ingress is exposed externally or not.

**Review by wbreza — APPROVED** [review]
Looks good - added a couple minor comments.

File: templates/common/infra/bicep/core/security/keyvault-secrets.bicep
Comment (jongio) [reply templates/common/infra/bicep/core/security/keyvault-secrets.bicep:9]: Removed the file

**Comment by jongio** [comment]
Waiting on a test pass before we merge.

**Pull request merged**
